src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpEnvironmentProxy.cs

```diff
@@ -15,6 +15,7 @@ internal sealed class HttpEnvironmentProxyCredentials : ICredentials
         private readonly NetworkCredential? _httpsCred;
         private readonly Uri? _httpProxy;
         private readonly Uri? _httpsProxy;
+        internal static readonly string s_defaultCredentialsPlaceholder = "$";
```

Comment:
error CA1802: Field 's_defaultCredentialsPlaceholder' is declared as 'readonly' but is initialized with a constant value. Mark this field as 'const' instead.